### PR TITLE
Enable Readium advanced settings for ePubs

### PR DESCRIPTION
**Pull request by mauricecarrier7**
**What's this do?**
Enabling Readium advanced options to ensure font size is adjusted when changed in settings.

**Why are we doing this? (w/ Notion link if applicable)**
https://www.notion.so/lyrasis/Text-resizing-is-not-working-for-an-Axis360-title-on-iOS-9869f8694bc64ec086d4b15fd8ea3fa5

https://readium.org/readium-css/docs/CSS12-user_prefs.html#:~:text=the%20required%20flag.-,Font%20size,Required%20flag%3A%20%2D%2DUSER__advancedSettings%3A%20readium%2Dadvanced%2Don,-Override%20class%3A%20User

**How should this be tested? / Do these changes have associated tests?**
Follow recreation steps in ticket

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new Palace build for QA?**
Yes

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@mauricecarrier7 

https://user-images.githubusercontent.com/6921353/187925470-01c12554-33cd-4069-a6ab-ee1ecfe7c78b.MP4

### CODE REVIEW

File: Palace.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4429,9 +4433,10 @@
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 101;
-				DEVELOPMENT_TEAM = 88CBA74T8K;
+				CODE_SIGN_IDENTITY = "Apple Development";
```
Comment (vladimirfedorov) [inline Palace.xcodeproj/project.pbxproj:4436]: We can't use this, `fastlane` needs `iPhone Distribution` with `Ad hoc 2` profile

**Review by vladimirfedorov — CHANGES_REQUESTED** [review]
`project.pbxproj` has conflicting provisioning settings

File: Palace.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mauricecarrier7) [reply Palace.xcodeproj/project.pbxproj:4436]: Ah, meant to revert this, thanks.

**Review by vladimirfedorov — APPROVED** [review]
Looks good to me!

**Pull request merged**